packages/next-on-pages/templates/_worker.js/utils/matcher.ts

```diff
@@ -13,34 +17,64 @@ type HasFieldRequestProperties = {
  */
 export function hasField(
```

Comment:
Given the new return type I think the name of this function needs to be updated 🤔 
```suggestion
export function checkHasField(
```
(or something along these lines)

Reply:
Also, could you please update the tsdoc comment?

the function description can clarify that performs a check and returns the new destination (if it exists) also the return says ``* @returns Whether the request matches the `has` record conditions.`` which is no longer totally correct